components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -68,8 +70,19 @@ export default function MeasureHighlightingPanel({ patientId }: MeasureHighlight
         limit={100}
         label="Search CQL Expression Definition"
         onItemSubmit={item => {
-          document.getElementById(defIds[item.value])?.scrollIntoView({ behavior: 'smooth' });
+          document
+            .querySelector(`[data-statement-name="${item.value.split('"')[1]}"`)
```

Comment:
I don't think we need to since it'll look for that string, but I can add it if you think it's necessary?

Reply:
The query selector we really want to create is: `pre[data-statement-name="statement Name"][data-library-name="library name"]`

Adding the `pre` on the front makes it more clear what we are looking for and potentially faster to. We definitely need the closing square brackets to make it a valid CSS query selector since some browsers may choose to completely throw out selectors that have any parsing issues.